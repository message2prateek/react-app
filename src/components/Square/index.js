import styled from 'styled-components'

const Square = styled.span`
  color: Crimson;
  font-size: 16vh;
  font-weight: bold;
  line-height: 20vh;
  text-align: center;
  text-transform: uppercase;
`

Square.displayName = 'Square'

export default Square
