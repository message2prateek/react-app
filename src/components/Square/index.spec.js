import * as React from 'react'

import { shallow } from 'enzyme'

import Square from './'

describe('Square', () => {
  it('matches the snapshot', () => {
    expect(
      toJson(
        shallow(<Square />)
          .first()
          .render()
      )
    ).toMatchSnapshot()
  })
})
