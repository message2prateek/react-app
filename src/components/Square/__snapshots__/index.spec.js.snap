// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Square matches the snapshot 1`] = `
.c0 {
  color: Crimson;
  font-size: 16vh;
  font-weight: bold;
  line-height: 20vh;
  text-align: center;
  text-transform: uppercase;
}

<span
  class="c0"
/>
`;
